spec/index.bs

```diff
@@ -2655,16 +2655,16 @@ dictionary GPUSamplerBindingLayout {
 </dl>
 
 <script type=idl>
-enum GPUTextureType {
+enum GPUTextureSampleType {
   "float",
   "unfilterable-float",
-  "depth-float",
+  "depth",
   "sint",
   "uint",
 };
 
 dictionary GPUTextureBindingLayout {
-    GPUTextureType type = "float";
+    GPUTextureSampleType type = "float";
```

Comment:
Should we call this field `sampleType` or something?

Reply:
That would break the pattern of every other binding layout member having a `type`, which is a nice bit of consistency. FWIW We currently have the `type` member of `GPUStorageTextureBindingLayout` be a `GPUStorageTextureAccess` enum, so it's already not a 1:1 enum/member name mapping.